vendor-bin/phpunit/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require-dev": {
+        "phpunit/phpunit": "^9.5.26",
```

Comment:
PHPUnit runs in the same process than our code so it cannot have different dependencies.

Reply:
It will work fine, external tools and their dependencies should not affect application/lib dependencies. Had many occasions that phpunit dependency example symfony/process conflicts with app/lib dependencies and had to upgrade phpunit first.

Reply:
That's the reason they must be compatible: when running tests, PHPUnit will run the tested code inside its own process. And as PHP cannot load 2 definitions of a class, your code will run against the Process class of the PHPUnit dependency instead of using its own dependency, which will break.

Isolating tools is possible when the tool does not execute your own code in its process.

Reply:
Right , phpunit is probably an exception , other tools like psalm could be moved as they don't autoload code

Reply:
And that's why I reported the issue for PHPUnit, not for Psalm

Reply:
reverted phpunit.